sawmill-core/src/main/java/io/logz/sawmill/processors/XmlProcessor.java

```diff
@@ -128,6 +129,7 @@ public ProcessResult process(Doc doc) {
 
             xmlNodes.compute(key, (k, oldVal) -> {
                 if (node.getChildNodes().item(0) == null && omitEmptyTagAsFieldEnabled) { return oldVal; }
+                if (node.getChildNodes().item(0) == null && !omitEmptyTagAsFieldEnabled) { return "null"; }
```

Comment:
shouldn't you return `null` and not `"null"` ?

Reply:
I thought so too, it's already null when the code is getting there but let's talk about it, I want you to see what value appears at the end when you take the object

Reply:
k let's simulate it tomorrow together